### DISCUSSION TITLE
Broken screenshot for scrollable containers with overflow

### DISCUSSION THREAD

**Discussion by patrickschroeter** [FRSOURCE/cypress-plugin-visual-regression-diff#217] [Q&A]
When using snapshots for Views which are larger than the page or are scrollable the resulting snapshot is not correct. 

Does anyone has similar issues of do I only miss a configuration?

![AppComponent should match snapshot #0 actual](https://user-images.githubusercontent.com/14996876/221855629-830f1674-41fc-47f7-b145-b8f2257d9036.png)

**Comment by FRSgit** [comment]
Hey @patrickschroeter!

Is the image above a single screenshot made by the plugin itself?
Can you provide some simple reproduction scenario? You can example directory from this repo as a started for reproduction: https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff/tree/main/example

**Reply by patrickschroeter** [reply]
@FRSgit Thanks for your answer. I was able to reproduce it by changing the scroll container from `html` to `body` by adding the following CSS to the `App.vue` in the example project has the same effect:
```
html {
  height: 100%;
  overflow: hidden;
}

body {
  height: 100%;
  overflow: auto;
}
```

![Screenshot 2023-03-30 at 10 45 24](https://user-images.githubusercontent.com/14996876/228781391-89f0c10e-0500-4f6b-bda2-6bbe1b76222a.png)


Tested on macOS with Plugin Version 3.2.14

**Reply by patrickschroeter** [reply]
@FRSgit Were you able to reproduce this issue or is it still a unique problem? Should I create an issue out of it?
